### ISSUE TITLE
How to debug ts with devtools?

### ISSUE THREAD

**Issue by chouchoui** [vhanla/vuets#1]
![snipaste_2018-09-10_16-40-41](https://user-images.githubusercontent.com/14866249/45286390-49fd9c00-b518-11e8-954d-75c7d8d33bbb.png)
Can't find ClientApp folder in dist folder

**Comment by mzhukovs** [comment]
Yes if I set any breakpoints in the ts files in VS2017 none of them get hit: "breakpoint set but not yet bound" - did you have any luck @chouchoui ?

**Comment by NPadrutt** [comment]
same issue here. I checked the vue template in the Microsoft.AspNetCore.SpaTemplates package which is the foundation for this here and that works without any issue.

**Comment by NPadrutt** [comment]
I have to correct my post. The issue wasn't the loader but the mode flag. When I removed the following line it works for me with both loaders. When I readd it, it doesn't with both.

`mode: devMode`

Is it possible that this is because development mode sets the devtool to eval? 

EDIT 2: after some more testing I have to say  it seems it's not due to the flag above. Rather it seems that it generates a build of the typescript classes from time to time (also when you change that flag) . I then have to delete all of these and then it seems to work again.. BuildOnSave is set to false in my tsconfig.

**Comment by yukihiro-hyakuta** [comment]
I have some problem to degug with ts.
I am trying and resolution
comment out below

webpack.config.js
//mode:devMode,


`const path = require('path');
const webpack = require('webpack');
const ExtractTextPlugin = require('extract-text-webpack-plugin');
const bundleOutputDir = './wwwroot/dist';
const VueLoaderPlugin = require('vue-loader/lib/plugin');
const UglifyJsPlugin = require('uglifyjs-webpack-plugin');

module.exports = (env) => {
    const isDevBuild = !(env && env.prod);
    const devMode = isDevBuild ? 'development' : 'production';

    return [{
        //mode: devMode,
        stats: { modules: false },
        context: __dirname,
        resolve: { extensions: ['.js', '.ts'] },`
